services/src/test/scala/cromwell/pipeline/service/ProjectConfigurationServiceTest.scala

```diff
@@ -22,72 +22,43 @@ class ProjectConfigurationServiceTest extends AsyncWordSpec with Matchers with M
   private val projectConfigurationId = ProjectConfigurationId.randomId
 
-  private val projectFileConfiguration: ProjectFileConfiguration =
-    ProjectFileConfiguration(Paths.get("/home/file"), List(FileParameter("nodeName", StringTyped(Some("hello")))))
+  private val wdlParams: WdlParams =
+    WdlParams(Paths.get("/home/file"), List(FileParameter("nodeName", StringTyped(Some("hello")))))
 
   private val activeConfiguration: ProjectConfiguration =
     ProjectConfiguration(
       projectConfigurationId,
       projectId,
       active = true,
-      List(projectFileConfiguration),
+      wdlParams,
       ProjectConfigurationVersion.defaultVersion
     )
 
   private val inactiveConfiguration: ProjectConfiguration =
     activeConfiguration.copy(active = false)
 
-  private val updatedConfiguration: ProjectConfiguration = {
-    val updatedProjectFileConfiguration =
-      projectFileConfiguration.copy(inputs = List(FileParameter("nodeName", StringTyped(Some("hi")))))
-    activeConfiguration.copy(projectFileConfigurations = List(updatedProjectFileConfiguration))
-  }
-
   "ProjectConfigurationService" when {
     "add new configuration for project and user is project owner" should {
       val result: Unit = ()
       val error = new Exception("Oh no")
       when(projectService.getUserProjectById(projectId, userId)).thenReturn(Future.successful(dummyProject))
 
       "return success if creation was successful" in {
-        when(configurationRepository.getAllByProjectId(projectId)).thenReturn(Future.successful(Seq.empty))
         when(configurationRepository.addConfiguration(activeConfiguration)).thenReturn(Future.successful(result))
         configurationService.addConfiguration(activeConfiguration, userId).map(_ shouldBe result)
       }
 
-      "return success if update was successful" in {
-        when(configurationRepository.getAllByProjectId(projectId))
-          .thenReturn(Future.successful(Seq(activeConfiguration)))
-        when(configurationRepository.updateConfiguration(updatedConfiguration)).thenReturn(Future.successful(result))
-        configurationService.addConfiguration(updatedConfiguration, userId).map(_ shouldBe result)
-      }
-
-      "return failure if it couldn't fetch existing configuration" in {
-        when(configurationRepository.getAllByProjectId(projectId)).thenReturn(Future.failed(error))
-        configurationService.addConfiguration(activeConfiguration, userId).failed.map(_ shouldBe error)
-      }
-
       "return failure if creation wasn't successful" in {
-        when(configurationRepository.getAllByProjectId(projectId)).thenReturn(Future.successful(Seq.empty))
         when(configurationRepository.addConfiguration(activeConfiguration)).thenReturn(Future.failed(error))
         configurationService.addConfiguration(activeConfiguration, userId).failed.map(_ shouldBe error)
       }
-
-      "return failure if update wasn't successful" in {
```

Comment:
I think we should keep this test case

Reply:
It doesn't depend on the update method anymore. Which condition should be covered by this testcase?

Reply:
You are right, I haven't looked carefully. Closing this conversation.